content/Felles datakatalog/Search/Search/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Search
+weight: 1
+---
+
+The service responsible for handling searches is [fdk-search-service](https://github.com/Informasjonsforvaltning/fdk-search-service), which is described by [this OpenAPI specification](https://raw.githubusercontent.com/Informasjonsforvaltning/fdk-search-service/main/openapi.yaml).
+
+Production endpoint: <https://search.api.fellesdatakatalog.digdir.no/search>
+Demo endpoint: <https://search.api.demo.fellesdatakatalog.digdir.no/search>
+Staging endpoint: <https://search.api.staging.fellesdatakatalog.digdir.no/search>
+
+Simple example using the staging endpoint:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test"}'
+```
+
+### Boosting
+
+Hits from some fields will be prioritized over others, i.e. a matching hit from the title field will be prioritized over a hit from the description field
+
+### Pagination
+
+All search results will be paginated, it is possible to manipulate the size and page number with the pagination field in the search body.
+
+Example using the pagination field:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test","pagination":{"size":1,"page":5}}'
+```
+
+### Filtering
+
+It's possible to filter the search result, see SearchFilters in the OpenAPI specification for a list of all possible filters.
+
+Example using the data theme filter:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test","filters":{"dataTheme":{"value":["ENVI"]}}}'
+```
+
+Example using the open data filter:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test","filters":{"openData":{"value":true}}}'
```

Comment:
Har vi en binærverdi for åpne data? Hva er den basert på? Og hvordan spiller den sammen med disse filtrene? 

<img width="260" alt="image" src="https://github.com/user-attachments/assets/f8bd9ee8-2d99-4ee6-9da6-5f24afe28d4e">

Reply:
reasoning-service setter en binærverdi på om et datasett er åpne data eller ikke

Reply:
ett av kravene som den bruker er at datasettet har allmenn tilgang, så alle datasett som er type åpne data har allmenn tilgang, men ikke alt med allmenn tilgang er åpne data

Reply:
Skjønner. Den informasjonen hadde det vært fint å fremme et sted, Kanskje på en av infosidene i katalogen, så man lettere forstår hva filtrene representerer.